src/components/panels/SpeakersPanel.jsx

```diff
@@ -7,56 +7,65 @@ import {
   EuiText,
 } from "@elastic/eui";
 import React from "react";
-import { speakers } from "../../data/speakers";
+import { useEffect } from "react";
+// import { speakers } from "../../data/speakers";
```

Comment:
Can we remove this comment and instead do something like 

// // speaker data lives in this spreadsheet: https://docs.google.com/spreadsheets/d/1XgyHXaReTZ3Nq_r7QS18GDvqK_ht010QqnI6PXAnePA/edit#gid=0

Reply:
Sure